.travis.yml

```diff
@@ -15,7 +15,10 @@ dist: trusty
 env:
   global:
     - ERT_SHOW_BACKTRACE=1
-    - LD_LIBRARY_PATH="$(pwd)/install/lib64"
+    - INSTALL_DIR="$(pwd)/install"
+    - LD_LIBRARY_PATH="${INSTALL_DIR}/lib:${INSTALL_DIR}/lib64"
+    - PYTHONPATH="${INSTALL_DIR}/lib/python2.7/site-packages:${INSTALL_DIR}/lib64/python2.7/site-packages:$PYTHONPATH"
```

Comment:
> Found ecl.  2.4.git >= 2.1.0 in /home/travis/build/equinor/ert/install/lib/python3.7/dist-packages

I'm a bit confused here, because this does not appear to be part of what we inject in the `PYTHONPATH`?